pkg/jx/cmd/get_secret.go

```diff
@@ -0,0 +1,82 @@
+package cmd
```

Comment:
This isn't well thought-out at the moment and does a bad job. Don't worry about it at the moment. Will fix later.